docs/sources/enterprise/access-control/permissions.md

```diff
@@ -58,6 +58,14 @@ The following list contains fine-grained access control actions.
 | `org.users.role:update`          | `users:*` <br> `users:id:*`                                                                 | Update the organization role (`Viewer`, `Editor`, or `Admin`) of an organization.                                                                          |
+| `orgs:read`                      | `orgs:*` <br> `orgs:id:*`                                                                   | Read organization.                                                                                                                                         |
+| `orgs:write`                     | `orgs:*` <br> `orgs:id:*`                                                                   | Update organization.                                                                                                                                       |
+| `org:create`                     | n/a                                                                                         | Create organization.                                                                                                                                       |
+| `orgs:delete`                    | `orgs:*` <br> `orgs:id:*`                                                                   | Delete organization.                                                                                                                                       |
+| `orgs.quotas:read`               | `orgs:*` <br> `orgs:id:*`                                                                   | Read organization quotas.                                                                                                                                  |
+| `orgs.quotas:write`              | `orgs:*` <br> `orgs:id:*`                                                                   | Update organization quotas.                                                                                                                                |
+| `orgs.preferences:read`          | `orgs:*` <br> `orgs:id:*`                                                                   | Read organization preferences.                                                                                                                             |
+| `orgs.preferences:write`         | `orgs:*` <br> `orgs:id:*`                                                                   | Update organization preferences.                                                                                                                           |
 | `ldap.user:read`                 | n/a                                                                                         | Get a user via LDAP.                                                                                                                                       |
```

Comment:
Do we want to use `get` here, given we use `read` in the code? Consider using the same verb.